docusaurus.config.js

```diff
@@ -159,13 +159,7 @@ module.exports = {
       {
         containerId: 'GTM-N4NQJGX',
       },
-    ],
-    [
-      path.resolve(__dirname, './src/plugins/deity-intercom/src/index.js'),
-      {
-        appId: 'z91ewqiv',
-      },
-    ],
+    ]
```

Comment:
@hobadams why did you move the config to the component?

Reply:
@Borales - good point. I've moved it back into the config (but not as a plugin)